src/cocotb/_init.py

```diff
@@ -289,3 +291,22 @@ def _setup_regression_manager() -> None:
     elif test_filter_str:
         cocotb.regression_manager.add_filters(test_filter_str)
         cocotb.regression_manager.set_mode(RegressionMode.TESTCASE)
+
+
+def _setup_resolve_x() -> None:
+    envvar = os.getenv("COCOTB_RESOLVE_X", None)
+    try:
+        resolver = cast(
+            Union[ResolverLiteral, None],
+            {
+                None: None,
+                "VALUE_ERROR": None,
+                "ZEROS": "zeros",
+                "ONES": "ones",
+                "RANDOM": "random",
+            }[envvar],
+        )
```

Comment:
My Monkish brain dislikes the discrepancy between the all-caps keyword for the environment variable and the lowercase variant for the function argument. I guess we're not tanking performance if we lowercase the env var for backwards-compatibility (or accept the all-caps or the lowercase variant) and document the lower-case variant as the thing we want?

Reply:
> I guess we're not tanking performance if we lowercase the env var for backwards-compatibility (or accept the all-caps or the lowercase variant) and document the lower-case variant as the thing we want?

I had done that as part of a previous version of this PR, so I'll do that.